app/frontend/src/store/modules/apiAccess.js

```diff
@@ -0,0 +1,16 @@
+// Lots more to come here, not just the UI step...
+export default {
+  namespaced: true,
+  state: {
+    step: 1
+  },
+  getters: {
+    step: state => state.step
+  },
+  mutations: {
+    setStep: (state, step) => {
```

Comment:
i was just reading about passing objects to mutations in the store. presumably you can make a lot of the api calls from in this file too.

Reply:
That is correct, though by convention iirc, you'll want to try to do those calls in an action instead of a mutation since a mutation is synchronous and focuses on changing the state, whereas an action can be asynchronous, and is better suited for doing api calls that then call the mutations.